Description: disable use of distribute_setup, which bootstraps distribute installation
Author: Jakub Wilk <jwilk@debian.org>
Forwarded: not-needed
Last-Update: 2011-10-12

--- a/setup.py
+++ b/setup.py
@@ -2,6 +2,7 @@
 try:
     from setuptools import setup, find_packages
 except ImportError:
+    raise
     import distribute_setup
     distribute_setup.use_setuptools()
     from setuptools import setup, find_packages
